docs/section5/Rule5-18.md

```diff
@@ -18,26 +15,28 @@
 **Data Lookup:** Table G3.1.1-1  
 **Function Call:**  
 
-  1. get_building_segment_window_wall_areas()
+  1. get_area_type_window_wall_areas()
 
 ## Rule Logic:  
 
-- Get window wall areas dictionary for building: `window_wall_areas_dictionary_b = get_building_segment_window_wall_areas(B_RMR)`
+- Get window wall areas dictionary for building: `window_wall_areas_dictionary_b = get_area_type_window_wall_areas(B_RMR)`
 
 - For each building segment in the Baseline model: `for building_segment_b in B_RMR.building.building_segments:`
 
-  - Check if building segment area type is included in Table G3.1.1-1: `if data_lookup(table_G3_1_1_1, building_segment_b.area_type_vertical_fenestration):`
+  - Check if building segment area type is included in Table G3.1.1-1, save building segment area type to array if not already saved: `if ( building_segment_b.area_type_vertical_fenestration ) AND ( building_segment_b.area_type_vertical_fenestration NOT in area_type_array_b ): area_type_array_b.append(building_segment_b.area_type_vertical_fenestration)`
+
+- For each unique area type in building: `for area_type_b in area_type_array_b:`
 
-    - Calculate building segment window wall ratio: `building_segment_wwr_b = window_wall_areas_dictionary_b[building_segment_b.id][0] / window_wall_areas_dictionary_b[building_segment_b.id][1]`
+  - Calculate area type window wall ratio: `area_type_wwr = window_wall_areas_dictionary_b[area_type_b]["TOTAL_WINDOW_AREA"] / window_wall_areas_dictionary_b[area_type_b]["TOTAL_WALL_AREA"]`
 
-      **Rule Assertion:**
+    **Rule Assertion:**
 
-      - Case 1: If building is all new and building segment window-wall-ratio matches Table G3.1.1-1 allowance: `if ( B_RMR.building.is_all_new ) AND ( building_segment_wwr_b == data_lookup(table_G3_1_1_1, building_segment_b.area_type_vertical_fenestration) ): PASS`
+    - Case 1: For each area type in building, if building is all new and area type window-wall-ratio matches Table G3.1.1-1 allowance: `if ( B_RMR.building.is_all_new ) AND ( area_type_wwr == data_lookup(table_G3_1_1_1, area_type_b) ): PASS`
 
```

Comment:
is_all_new is now moved to the building_segment level. (I have this wrong in TDS as well.) This applies to this and other cases below.